### PR TITLE
refactor: move metadata into each of the sub-package `package.json` and add keywords

### LINKED ISSUE #848: Refactor sub-package `package.json` metadata
figure out some alternative to `package.json` generation (previously in esbuild.config.js)
- either some post-build script, 
- or just move this metadata into the source `package.json`s

Note: moving the metadata into each of the sub-packages seems like the best approach, we can leverage custom `keywords`, `bugs` issues link and other customizations for each of the sub-packages

### PR DESCRIPTION
**Pull request by vmasek**
closes #848 

- [x] remove esbuild post build step transforming sub-package `package.json`s
- [x] move all necessary metadata into each of the sub-package `package.json`s
- [x] add keywords to all sub-package `package.json`s
  - NPM displays keywords and uses them for search categorizing, I've added a few so our package is more visible

### CODE REVIEW

**Review by matejchalk — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -126,6 +126,27 @@
     "@nx/nx-darwin-x64": "^16.10.0",
     "@nx/nx-linux-x64-gnu": "16.7.4"
   },
+  "keywords": [
```
[the remark this answers is not in the document]
Comment (vmasek) [reply package.json:129]: Thanks, I suspected I need to copy it to individual ones, will do.

File: packages/nx-plugin/package.json [hunk 2]
```diff
@@ -6,26 +6,37 @@
   "publishConfig": {
     "access": "public"
   },
-  "homepage": "code-pushup.dev",
+  "homepage": "https://code-pushup.dev",
   "bugs": {
-    "url": "https://github.com/code-pushup/cli/issues"
+    "url": "https://github.com/code-pushup/cli/issues?q=is%3Aissue%20state%3Aopen%20type%3ABug%20label%3A\"🧩%20nx-plugin\""
```
[the remark this answers is not in the document]
Comment (vmasek) [reply packages/nx-plugin/package.json:11]: I went ahead and marked all open & closed issues with the `Bug` type and I checked it displays correctly even in a private tab so I'd say it should be good. I can have a look if the templates support it now and if no we anyway need to acknowledge these bugs in the processing column

File: packages/plugin-coverage/package.json [hunk 3]
```diff
@@ -2,6 +2,39 @@
   "name": "@code-pushup/coverage-plugin",
   "version": "0.52.0",
   "description": "Code PushUp plugin for tracking code coverage ☂",
+  "license": "MIT",
+  "homepage": "https://github.com/code-pushup/cli/tree/main/packages/plugin-coverage#readme",
+  "bugs": {
+    "url": "https://github.com/code-pushup/cli/issues?q=is%3Aissue%20state%3Aopen%20type%3ABug"
```
[the remark this answers is not in the document]
Comment (vmasek) [reply packages/plugin-coverage/package.json]: done

File: packages/nx-plugin/package.json [hunk 4]
```diff
@@ -6,26 +6,37 @@
   "publishConfig": {
     "access": "public"
   },
-  "homepage": "code-pushup.dev",
+  "homepage": "https://code-pushup.dev",
   "bugs": {
-    "url": "https://github.com/code-pushup/cli/issues"
+    "url": "https://github.com/code-pushup/cli/issues?q=is%3Aissue%20state%3Aopen%20type%3ABug%20label%3A\"🧩%20\""
```
Comment (matejchalk) [inline packages/nx-plugin/package.json:11]:
```suggestion
    "url": "https://github.com/code-pushup/cli/issues?q=is%3Aissue%20state%3Aopen%20type%3ABug%20label%3A\"🧩%20nx-plugin\""
```

**Review by matejchalk — APPROVED** [review]

**Pull request merged**